server/src/test/scala/za/co/absa/atum/server/aws/AwsSecretsProviderSpec.scala

```diff
@@ -47,7 +47,7 @@ class AwsSecretsProviderSpec extends ConfigProviderSpec {
           awsConfig <- ZIO.config[AwsConfig](AwsConfig.config)
           awsSecretValue <- AwsSecretsProvider.getSecretValue(awsConfig.dbPasswordSecretName)
         } yield assertTrue(dummySecretValue == awsSecretValue)
-      }
+      } @@ TestAspect.tag("IntegrationTest")
```

Comment:
Again, I think it's not an integration test. 🤔

Reply:
Tags reduced.